packages/components/src/components/tab-nav/tab-nav.tsx

```diff
@@ -146,22 +150,30 @@ export class TabNav {
 
   linkPanels() {
     const tabs = this.getAllTabs();
-    const selectedTab =
-      tabs.find((x) => x.selected) || tabs.filter((x) => !x.disabled)[0];
-
```

Comment:
i'm still thinking this functionalyty must be in separate method, it took me some time to find it here when i started

Reply:
i'm not sure what you mean, i put this stuff back into linkPanels as separate call and renamed linkPanels to   linkPanelsAndSelectTab. or how you see it?

Reply:
you can make this a separate function because this also gets called in other places. and they can both reference this function 
```js
const selectTab = (tabs) =>  tabs.find((x) => x.selected) || tabs.filter((x) => !x.disabled)[0];
```

Reply:
now understood, hehe